.travis.yml

```diff
@@ -17,7 +17,6 @@ install:
   - node --version
   - npm version
   - nvm version
-  - bundle install --jobs=3 --retry=3
```

Comment:
the issue seem to be `rubocop-rails`, removing that [dependency line](https://github.com/theforeman/foreman_rh_cloud/blob/master/foreman_rh_cloud.gemspec#L25) fixed it for me

Reply:
Interesting... Doesn't make any sense though.